src/esp/assets/ResourceManager.cpp

```diff
@@ -325,8 +325,10 @@ bool ResourceManager::loadStage(
     esp::scene::SceneManager* sceneManagerPtr,
     std::vector<int>& activeSceneIDs) {
   // If the semantic mesh should be created, based on SimulatorConfiguration
-  const bool createSemanticMesh =
-      metadataMediator_->getSimulatorConfiguration().loadSemanticMesh;
+  // todo sangarg : Fix this, the config.json file seems to be incorrect
```

Comment:
metadataMediator_->getSimulatorConfiguration().loadSemanticMesh always returns false for me
Not sure if my scene config file is correct. Any idea how to debug this?

Reply:
That code path has changed a bit recently, be sure to merge latest `main` into your branch. Then use the same assets CI is using to check. MP3D example should work, but download fresh because we've updated the config in the last couple months.